cmd/project.go

```
@@ -821,6 +821,39 @@
+		}
+	}
+	tui.ShowSpinner("fetching projects ...", action)
+	if len(projects) == 0 {
+		showNoProjects()
+		return ""
+	}
+	var options []tui.Option
+	for _, p := range projects {
+		desc := p.Description
+		if desc == "" {
+			desc = emptyProjectDescription
```

golangci-lint (1.64.8)

ineffectual assignment to desc

(ineffassign)